New featuress/2nd new feature

```diff
@@ -0,0 +1,3 @@
+This is my 2nd New feature // Madhu
+
+New line for 2nd New Feature
```

Comment:
Its really good that you have decided to add this  line of code

Reply:
Thanks